.travis.yml

```diff
@@ -20,20 +17,20 @@ install:
 # command to run the dependencies
 script:
   - 'if [[ -n "$RUN_INTEGRATION_TESTS" ]]; then rm -fr ~/.cache/pip; fi'
-  - "pipenv run bash ./run-tests.sh"
+  - 'export PYPI_VENDOR_DIR="$(pwd)/tests/pypi/"'
+  - 'pip install -e "$(pwd)" --upgrade'
+  - 'pipenv install --deploy --system --dev'
+  - 'pipenv run time pytest -v -n 4 -m "$TEST_SUITE" tests --tap-stream'
 
 jobs:
   include:
-    - stage: integration
-      env: TEST_SUITE='cli'
     - stage: takes-forever
       env:
         - TEST_SUITE='install'
         - PYTEST_ADDOPTS='--cache-clear'
         - RUN_INTEGRATION_TESTS=1
```

Comment:
Overall you can just add here so the section looks like this:

```yml
jobs:
  include:
    - stage: takes-forever
      python: "3.6"
      env:
        - TEST_SUITE='install'
        - CACHE_NAME='INSTALL-1'
        - PYTEST_ADDOPTS='--cache-clear'
        - RUN_INTEGRATION_TESTS=1
    - stage: takes-forever
      python: "2.7"
      env:
        - CACHE_NAME='INSTALL-2'

Reply:
I couldn't get the above to work for some reason but I have no idea why, it seems like it should work fine to me... so maybe scratch that?